### DISCUSSION TITLE
Coleman Palm Springs appears to have been updated to the square pump

### DISCUSSION THREAD

**Discussion by Stratoco** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#601] [General]
Just ordered one off of amazon on a flash sale. Going from the pics and manual, it appears to have been updated to model # S100105. 

Link: https://www.amazon.com/Coleman-SaluSpa-Inflatable-Hot-Tub/dp/B00NB3P98G/ref=cm_cr_arp_d_product_top?ie=UTF8&th=1

Will provide confirmation when it arrives. 

I do have a question though: has anyone managed to get a working temp override? I have had another saluspa model previously and the 104F limit just doesn't do the trick for me, especially since the pump/heater struggles to maintain temp when you're actually using it.

**Comment by wyliecoyoteuk** [comment]
In fact the spa probably reaches as high as 42C (108F), due to the 2C hysteresis of the sensor, but the display will not show higher than 40C(104F).

> Spa temperatures over 40 degrees Celsius (104F) can be dangerous to your health and safety. It can lead to high body temperature, dehydration or hyperthermia. It can also lead to skin rashes, heatstroke, cramps and exhaustion.

**Reply by Stratoco** [reply]
Spare me. 

The temperature of the water in my home comes out of the water heater at a nice steamy 160F, and is delivered to my bath closer to 140. By the time my tub is filled, it’s sitting at 115F - the perfect temperature. 

Even Cleveland Clinic docs have said that 112F is perfectly suitable. 

Just because a couple of old overweight drunks got dehydrated and died doesn’t mean everyone else should be punished. Use common sense.

**Reply by wyliecoyoteuk** [reply]
Just quoting advice from a UK medical site (with Fahrenheit conversions added) no need to get upset.
Obviously Americans are more heat tolerant, maybe because they use an antiquated temperature scale.:-)
Anyway, overriding the built in thermostat would involve physical modification of the pump, so not really possible using VA's hack.
Oh, and the newer versions have an improved sensor with a 1C hysteresis, so max 41C I'm afraid.

**Reply by Stratoco** [reply]
Got it. That’s what I had figured. Might have to put a weekend aside for that at some point, was just hoping there would be an easier way.
